plugins/paper/src/main/java/com/playmonumenta/plugins/custominventories/ClassSelectionCustomInventory.java

```diff
@@ -40,12 +40,21 @@ public class ClassSelectionCustomInventory extends CustomInventory {
 	private static final int P1_RESET_CLASS_LOC = 47;
 	private static final int P1_RESET_SPEC_LOC = 49;
 	private static final int P1_CHANGE_TRIGGERS_LOC = 51;
-	public static final ArrayList<Integer> P2_ABILITY_LOCS = new ArrayList<>(Arrays.asList(10, 14, 19, 23, 28, 32, 37, 41));
+	public static final ArrayList<Integer> P2_ABILITY_LOCS = new ArrayList<>(Arrays.asList(9, 14, 18, 23, 27, 32, 36, 41));
```

Comment:
Don't want to worsen the vanilla experience in support of modded here, better to change the RP to have a seperate background for the non-r3 version, since using an r3 background on the r1/2 version is going to make it both look on with and without the RP